packages/mdc-dialog/README.md

```diff
@@ -336,6 +335,12 @@ Mixin | Description
 
 > *NOTE*: The `max-width` and `max-height` mixins only apply their maximum when the viewport is large enough to accommodate the specified value when accounting for the specified margin on either side. When the viewport is smaller, the dialog is sized such that the given margin is retained around the edges.
 
+## Other Customizations
+Data Attributes | Description
+--- | ---
+`data-mdc-dialog-button-default` | Optional. Add to a button to indicate that it is the default action button (see Default Action Button section above).
+`data-mdc-dialog-element-focus` | Optional. Add to an element to indicate that it is the element to initially focus on after the dialog has opened. This is the element that should be passed in to `Adapter#getInitialFocusEl`.
```

Comment:
Remove __This is the element that should be passed in to `Adapter#getInitialFocusEl`.__ since it is framework wrapper specific.